dll/win32/ieframe/lang/ro-RO.rc

```diff
@@ -42,8 +48,8 @@ IDR_BROWSE_MAIN_MENU MENU
 
 STRINGTABLE
 {
-    IDS_INTERNET "Navigatorul de internet"
-    IDS_INTERNET_DESCRIPTION "Opens a Web browser and displays information on the Internet."
+    IDS_INTERNET "Navigator de Internet"
```

Comment:
Since you updated this string, please also update it in desk.cpl (`IDC_ICONS_INTERNET`), see https://git.reactos.org/?p=reactos.git&a=search&h=00420aff1f143467e7a6b4aebb31e1f86ec0e537&st=grep&s=Navigatorul+de+internet

Reply:
I will rename as Navigatorul de internet